pom.xml

```diff
@@ -72,23 +90,24 @@
         <dependency.spock.version>1.3-groovy-2.4</dependency.spock.version>
         <junit.version>4.13.2</junit.version>
         <trimStackTrace>false</trimStackTrace>
+        <liquibase-pro.version>0-SNAPSHOT</liquibase-pro.version>
```

Comment:
This doesn't appear to be used anywhere, and if it is not, it should be removed.

Reply:
I think we are using it in build.yml in step:  Publish tar.gz package to GPM